Translate accessibility.md to Portuguese (#1336)
HTTPArchive/almanac.httparchive.org: src/content/pt/2019/accessibility.md

---
part_number: II
chapter_number: 9
title: Acessibilidade
description: Capítulo de acessibilidade do Web Almanac de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.
authors: [nektarios-paisios, obto, kleinab]

> reviewers: [ljme]

Reviewer: Add Analysts added in #1334 

```suggestion
reviewers: [ljme]
analysts: [dougsillars, rviscomi, obto]
```

---
part_number: II
chapter_number: 9

> title: Accessibility

Reviewer: Oh one more:

```suggestion
title: Acessibilidade
```

> description: Accessibility chapter of the 2019 Web Almanac covering ease of reading, media, ease of navigation, and compatibility with assistive technologies.

Reviewer: You seem to have taken @HakaCode 's feedback too far 😀 He was just saying keep the "Web Almanac" brand name in English but translate the rest of this sentence.

So something like this?

```suggestion
description: Capítulo de acessibilidade do da Web Almanac de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.
```

Por causa disso, os desenvolvedores têm a capacidade de desativar esse recurso definindo uma das duas propriedades a seguir na [meta tag viewport](https://developer.mozilla.org/pt-BR/docs/Mozilla/Mobile/Viewport_meta_tag):

1. `user-scalable` definido como `0` ou `no`

2. `maximum-scale` definida como `1`, `1.0`, etc

Infelizmente, os desenvolvedores abusaram tanto disso que quase um em cada três sites no celular (32,21%) desabilita esse recurso e a Apple (a partir do iOS 10) não permite mais que os desenvolvedores da web desabilitem o zoom. O Safari para dispositivos móveis simplesmente [ignora a tag](https://archive.org/details/ios-10-beta-release-notes). Todos os sites, não importa o quê, podem ser ampliados e dimensionados em dispositivos iOS mais recentes.

{{ figure_markup(
  image="fig2.png",
  caption="Porcentagem de sites que desativam o zoom e o dimensionamento em comparação ao tipo de dispositivo.",

>   description="Dados de porcentagem de medição vertical, variando de 0 a 80 em incrementos de 20, vs. o tipo de dispositivo, agrupado em desktop e celular. Desktop habilitado: 75,46%; Desktop desabilitado 24,54%; Mobile habilitado: 67,79%; Celular desabilitado: 32,21%.",

Author: just changed 'Dabilitado' to 'Desabilitado'

Apesar disso, apenas 4,32% das páginas com tabelas apresentam legendas.

## Compatibilidade com tecnologias assistivas

### O uso de ARIA

Uma das especificações mais populares e amplamente utilizadas para acessibilidade na web é o padrão [Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/) (ARIA). Este padrão oferece uma grande variedade de atributos HTML adicionais para ajudar a transmitir o propósito por trás dos elementos visuais (ou seja, seu significado semântico) e de quais tipos de ações eles são capazes.

Usar o ARIA de maneira correta e apropriada pode ser desafiador. Por exemplo, das páginas que utilizam atributos ARIA, encontramos 12,31% com valores inválidos aos seus atributos. Isso é problemático porque qualquer erro no uso de um atributo ARIA não tem efeito visual na página. Alguns desses erros podem ser detectados com o uso de uma ferramenta de validação automatizada, mas geralmente exigem o uso prático de software de assistência real (como um leitor de tela). Esta seção examinará como o ARIA é usado na web e, especificamente, quais partes do padrão são mais predominantes.

> 

Reviewer: Remove this extra line

```suggestion
```

Author: I have read the English version again and re-translated. @HakaCode Can you check if it sounds better? It is in my last commit.

Reviewer: Think this comment is under the wrong bit of feedback.

---
part_number: II
chapter_number: 9
title: Acessibilidade

> description: Capítulo de acessibilidade do Almanaque da Web de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.

Reviewer 2: >description: Capítulo de acessibilidade do Almanaque da Web de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.

@eduqg de acordo com [Translators'-Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide), acredito que não possa ser traduzida.

>English product names (Web Almanac, HTTP Archive) should remain in English but can be adjusted slightly to local version (e.g. Le Web Almanac rather than The Web Almanac). For some languages this rule may be ignored if there are language specific character sets (e.g. Japanese might use katakana, hiragana, or kanj rather than English if that makes more sense).

authors: [nektarios-paisios, obto, kleinab]
reviewers: [ljme]
translators: [edu_qg]
discuss: 1764
results: https://docs.google.com/spreadsheets/d/16JGy-ehf4taU0w4ABiKjsHGEXNDXxOlb__idY8ifUtQ/
queries: 09_Accessibility
published: 2019-11-11T00:00:00.000Z
last_updated: 2020-09-21T00:00:00.000Z
---

## Introdução

> A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou não recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar em tecnologias assistivas que os usuários confiam.

Reviewer: @HakaCode had the folllowing in the `featured_chapter.html` file.

I'll leave you two to decide which is the better wording, but we should be consistent as it's supposed to be a quote from this chapter.

```suggestion
A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou omitir recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar a utilização de tecnologias assistivas.
```

Reviewer 2: I liked the suggestion.

Reviewer: So what have we decided here? Change the chapter inline with the `src/templates/pt/2019/featured_chapters.html` quote? Or the other way around?

@eduqg can you do one or the other?

Author: I have read the English version again and re-translated. @HakaCode  Can you check if it sounds better? It is in my last commit.

Reviewer 3: Hi, @eduqg I read your re-translation but we need to keep the same translation at the featured_chapter. I'm going to let here my suggested translation based on what you have already done (full paragraph). Then you can choose how to adapt:

>A acessibilidade na web é essencial para uma sociedade inclusiva e igualitária. À medida que mais de nossas vidas sociais e profissionais mudam para o mundo online, torna-se ainda mais importante para as pessoas com deficiência poderem participar de todas as interações online sem barreiras. Assim como os arquitetos de edifícios podem criar ou não recursos de acessibilidade, como rampas para cadeiras de rodas, os desenvolvedores da web podem ajudar ou atrapalhar na adoção de tecnologias assistivas de que os usuários dependem.

Na primeira tradução apontada, acho que tanto o "não" como o "omitir" cumprem o papel semântico ali, o segundo trecho foi que achei mais problemático.

### Contraste de cor

Existem muitos casos em que os visitantes do seu site podem não conseguir vê-lo perfeitamente. Os visitantes podem ser daltônicos e incapazes de distinguir entre a fonte e a cor de fundo ([1 em cada 12 homens e 1 em 200 mulheres](http://www.cvrl.org/people/stockman/pubs/1999%20Genetics%20chapter%20SSJN.pdf) de ascendência europeia). Talvez eles estejam simplesmente lendo enquanto o sol está alto e criando toneladas de brilho na tela - prejudicando significativamente a visão. Ou talvez eles simplesmente tenham envelhecido e seus olhos não consigam distinguir as cores tão bem como antes.

Para garantir que o seu site seja legível nessas condições, é fundamental que o texto tenha contraste de cor suficiente com o fundo. Também é importante considerar quais contrastes serão mostrados quando as cores forem convertidas para tons de cinza.

> 

Reviewer: Extra line has sneaked in here:

```suggestion
```

  width=568,
  height=300
  )
}}

Apenas 22,04% dos sites deram contraste de cor suficiente a todo o texto. Ou em outras palavras: 4 em cada 5 sites têm texto que se mistura facilmente com o fundo, tornando-o ilegível.

<p class = "note"> Observe que não foi possível analisar nenhum texto dentro das imagens, portanto, nossa métrica relatada é um limite superior do número total de sites que passaram no teste de contraste de cor. </p>

### Ampliando (zoom) e dimensionando páginas

Usando um [tamanho de fonte legível](https://accessibleweb.com/question-answer/minimum-font-size/) e [tamanho alvo](https://www.w3.org/WAI/WCAG21/quickref/#target-size) ajuda os usuários a ler e interagir com seu site. Mas mesmo os sites que seguem perfeitamente todas essas diretrizes não podem atender às necessidades específicas de cada visitante. É por isso que recursos do dispositivo, como a pinça para aplicar zoom e dimensionamento, são tão importantes: eles permitem que os usuários ajustem suas páginas para que suas necessidades sejam atendidas. Ou, no caso de sites particularmente inacessíveis que usam fontes e botões minúsculos, dá aos usuários a chance de usar o site.

> 

Reviewer: And here:

```suggestion
```

Existem raros casos em que a desativação do escalonamento é aceitável, como quando a página em questão é um jogo baseado na web que usa controles de toque. Se deixado ativado neste caso, os telefones dos jogadores irão aumentar e diminuir o zoom cada vez que o jogador tocar duas vezes no jogo, ironicamente tornando-o inacessível.

> Por causa disso, os desenvolvedores têm a capacidade de desativar esse recurso definindo uma das duas propriedades a seguir na [meta tag viewport](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag):

Reviewer: Can you check if Portuguese (either European or Brazilian) exist for all resources and use them if they do? Quite a few references to MDN in this chapter so only pointing out this first one but can you have a look at the rest?

```suggestion
Por causa disso, os desenvolvedores têm a capacidade de desativar esse recurso definindo uma das duas propriedades a seguir na [meta tag viewport](https://developer.mozilla.org/pt-BR/docs/Mozilla/Mobile/Viewport_meta_tag):
```

  caption="Popularidade dos níveis de título.",
  description="Gráfico de barras verticais medindo dados percentuais, variando de 0 a 80 em incrementos de 20, contra barras que representam cada nível de título h1 a h6. H1: 63,25%; H2: 67,86%; H3: 58,63%; H4: 36,38% ; H5: 14,64%; H6: 6,91%.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=1123601243&format=interactive"
  )
}}

### Marco principal

Um [marco principal](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role) indica aos leitores de tela onde o conteúdo principal de uma página da web começa para que os usuários possam ir direto. Sem isso, os usuários leitores de tela precisam pular manualmente sua navegação sempre que acessam uma nova página do site. Obviamente, isso é bastante frustrante.

Encontramos apenas uma em cada quatro páginas (26,03%) que inclui um marco principal. E, surpreendentemente, 8,06% das páginas continham erroneamente mais de um ponto de referência principal, deixando esses usuários adivinhando qual ponto de referência contém o conteúdo principal real.

> 

Reviewer: And here:

```suggestion
```

Todos esses elementos são projetados principalmente para suporte de acessibilidade e não têm efeito visual, o que significa que você pode substituir os elementos existentes por eles com segurança e não sofrer consequências indesejadas.

{{ figure_markup(
  image="fig5.png",
  caption="Uso de vários elementos semânticos HTML.",
  description="Gráfico de barras verticais com barras para cada tipo de elemento vs porcentagem de páginas que variam de 0 a 60 em incrementos de 20. nav: 53,94%; cabeçalho: 54,82%; rodapé: 55,92%; principal: 18,47%; à parte: 16,99% ; artigo: 22,59%; hr: 19,1%; seção: 36,55%.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=708035719&format=interactive"
  )
}}

### Outros elementos HTML usados ​​para navegação

Muitos leitores de tela populares também permitem que os usuários naveguem pulando rapidamente por links, listas, itens de lista, iframes e campos de formulário, como campos de edição, botões e caixas de listagem. A Figura 9.6 detalha a frequência com que vimos as páginas usando esses elementos.

> 

Reviewer: And here:

```suggestion
```

{{ figure_markup(
  image="fig6.png",
  caption="Outros elementos HTML usados ​​para navegação.",
  description="Gráfico de barras verticais com barras para cada tipo de elemento vs porcentagem de páginas que variam de 0 a 100 em incrementos de 25. a: 98,22%; ul: 88,62%; entrada: 76,63%; iframe: 60,39%; botão: 56,74% ; selecionar: 19,68%; área do texto: 12,03%.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=389034849&format=interactive"
  )
}}

> 

Reviewer: And here:

```suggestion
```
